### PR TITLE
Docs 7695 pm 2.3 troubleshooting enhancements mc

### PR DESCRIPTION
**Pull request by mcherem**

### CODE REVIEW

**Review by vijayang — APPROVED** [review]

File: modules/ROOT/pages/troubleshooting.adoc [hunk 1]
```diff
@@ -4,8 +4,12 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Partner Manager makes use of or depends on several different MuleSoft technologies, including but not limited to your Mule runtimes, Anypoint Runtime Manager, Anypoint Secrets Manager, and your infrastructure for B2B storage payloads.
+Anypoint Partner Manager makes use of or depends on several different MuleSoft technologies
```
Comment (kbostrom) [inline modules/ROOT/pages/troubleshooting.adoc]: "makes use of or depends on" is unnecessarily wordy:
```suggestion
Anypoint Partner Manager relies on several different MuleSoft technologies,
```

File: modules/ROOT/pages/troubleshooting.adoc [hunk 2]
```diff
@@ -4,8 +4,12 @@ ifndef::env-site,env-github[]
...
+including but not limited to Mule, Anypoint Runtime Manager, Anypoint Secrets Manager, and your infrastructure for B2B storage payloads.
```
Comment (kbostrom) [inline modules/ROOT/pages/troubleshooting.adoc]: Regardless of what your Devs say, you *must* follow the style ref. and term list:

"secrets manager, the secrets manager
Not a title or product. Never refer to as secrets vault, CSM, or the Secrets Manager."

```suggestion
including but not limited to Mule runtime engine (Mule), Anypoint Runtime Manager, the secrets manager, and your infrastructure for B2B storage payloads.
```

File: modules/ROOT/pages/troubleshooting.adoc [hunk 3]
```diff
@@ -4,8 +4,12 @@ ifndef::env-site,env-github[]
...
 
-Because Anypoint Partner Manager is used with other technologies, troubleshooting an issue with Anypoint Partner Manager often involves troubleshooting these other technologies.
+Because Anypoint Partner Manager is used with other technologies,
+troubleshooting an issue with Partner Manager often involves troubleshooting these other technologies.
```
Comment (kbostrom) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
troubleshooting an issue with Partner Manager often involves troubleshooting these other technologies as well.
```

File: modules/ROOT/pages/troubleshooting.adoc [hunk 4]
```diff
@@ -4,8 +4,12 @@ ifndef::env-site,env-github[]
...
 
-Use the topic links in the left navigation pane of the documentation to locate troubleshooting information specific to either Anypoint Partner Manager or its related technologies.
+Look for solutions here under the Configuration and Deployment, Connectivity or System, Data Processing error sections,
```
Comment (kbostrom) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
Look for solutions in documentation about connectivity and related errors,
```

File: modules/ROOT/pages/troubleshooting.adoc [hunk 5]
```diff
@@ -4,8 +4,12 @@ ifndef::env-site,env-github[]
...
+or use the topic links in the left navigation pane to locate troubleshooting information specific to either
```
Comment (kbostrom) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
or use documentation page's left navigation pane to locate troubleshooting information specific to either
```

File: modules/ROOT/pages/ts-config-deploy.adoc [hunk 6]
```diff
@@ -0,0 +1,8 @@
+= Troubleshooting Configuration and Deployment Related Errors
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-config-deploy.adoc]:
```suggestion
= Troubleshooting Configuration and Deployment-Related Errors
```

File: modules/ROOT/pages/ts-config-deploy.adoc [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If you encounter errors as you set up your trading partners and message flows,
+the messages and solutions in this section can help you resolve your issue.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-config-deploy.adoc]:
```suggestion
the messages and solutions described here can help you resolve your issue.
```

File: modules/ROOT/pages/ts-connectivity-system.adoc [hunk 8]
```diff
@@ -0,0 +1,7 @@
+= Troubleshooting Connectivity or System Related Errors
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-connectivity-system.adoc]:
```suggestion
= Troubleshooting Connectivity or System-Related Errors
```

File: modules/ROOT/pages/ts-connectivity-system.adoc [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If you encounter problems receiving or sending data, the messages and solutions in this section can help you resolve your issue.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-connectivity-system.adoc]:
```suggestion
If you encounter problems receiving or sending data, the messages and solutions described here can help you resolve your issue.
```

File: modules/ROOT/pages/ts-data-processing [hunk 10]
```diff
@@ -0,0 +1,7 @@
+= Troubleshooting Data Processing Related Errors
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-data-processing]:
```suggestion
= Troubleshooting Data Processing-Related Errors
```

File: modules/ROOT/pages/ts-data-processing [hunk 11]
```diff
@@ -0,0 +1,7 @@
...
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If you encounter problems with message validations and transformations, the messages and solutions in this section can help you resolve your issue.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-data-processing]:
```suggestion
If you encounter problems with message validations and transformations, the messages and solutions described here can help you resolve your issue.
```

File: modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc [hunk 12]
```diff
@@ -0,0 +1,21 @@
+= Deployment Initialization failed (with On Premise Deployment target)
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc:1]: Is this a verbatim error message; is that why it does not follow our guidelines for topic titles?

File: modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Causes
+
+* You do not have the appropriate server group configured or have the wrong name for the server group in your environment for the deployed message flow.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc]:
```suggestion
* You do not have the appropriate server group (or name for the server group) for the deployed message flow.
```

File: modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc [hunk 14]
```diff
@@ -0,0 +1,21 @@
...
+
+* Your runtime servers are either not set up or are not active.
+
+== Solution
+
+. Log in to Anypoint Platform > *Runtime Manager* in the same environment.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc]:
```suggestion
. In the deployment environment, log in to Anypoint Platform > *Runtime Manager*.
```

File: modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+
+. Click *Servers* and verify that you have the server group `B2BServerGroup` configured and is in `Running` Status.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc]:
```suggestion
. Click *Servers* and verify that the server group `B2BServerGroup` is configured and in `Running` status:
```

File: modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc [hunk 16]
```diff
@@ -0,0 +1,21 @@
...
+
+. If the server group is in a `Stopped` status contact your MuleSoft platform operations or administration team to review and restart the server.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc]: The next two steps are actually bullet-list options for rectifying the failure of the container step, and they should appear in the same order as in the container step:
```suggestion
* If `B2BServerGroup` is not set up, refer to https://docs.mulesoft.com/partner-manager/2.0/setup#setup[Set Up Anypoint Partner Manager].
* If the server group shows `Stopped` status, contact your MuleSoft platform operations or administration team to review and restart the server.
```

File: modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc [hunk 17]
```diff
@@ -0,0 +1,21 @@
...
+
+. If you do not have this server group setup, refer to https://docs.mulesoft.com/partner-manager/2.0/setup#setup[Setup Anypoint Partner Manager].
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail-onprem.adoc]: see above

File: modules/ROOT/pages/ts-deploy-initialize-fail.adoc [hunk 18]
```diff
@@ -0,0 +1,19 @@
+= Deployment Initialization failed
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail.adoc:1]: verbatim, I assume

File: modules/ROOT/pages/ts-deploy-initialize-fail.adoc [hunk 19]
```diff
@@ -0,0 +1,19 @@
...
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Cause
+
+Temporary System or connectivity issues between the Partner Manager control plane and the deployment server or (in case of CloudHub deployment) Anypoint MQ.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail.adoc]:
```suggestion
Your environment is experiencing temporary system or connectivity issues between the Partner Manager control plane and the deployment server or, in CloudHub deployment, Anypoint MQ. 
```

File: modules/ROOT/pages/ts-deploy-initialize-fail.adoc [hunk 20]
```diff
@@ -0,0 +1,19 @@
...
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Cause
+
+Temporary System or connectivity issues between the Partner Manager control plane and the deployment server or (in case of CloudHub deployment) Anypoint MQ. 
+
+== Solution
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail.adoc]:
```suggestion
== Solutions
```

File: modules/ROOT/pages/ts-deploy-initialize-fail.adoc [hunk 21]
```diff
@@ -0,0 +1,19 @@
...
+
+* Verify that your deployment server group *B2BServerGroup* is in the `Running` state.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail.adoc:13]: Is this correct? I'm looking for a way to make these options parallel:
```suggestion
* For CloudHub deployments, verify that your deployment server group *B2BServerGroup* is in the `Running` state.
```

File: modules/ROOT/pages/ts-deploy-initialize-fail.adoc [hunk 22]
```diff
@@ -0,0 +1,19 @@
...
+
+For On-Premises, try redeploying the message flow.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail.adoc]:
```suggestion
* For on-premises deployments, attempt to redeploy the message flow, wait 10 minutes, and try again.
```

File: modules/ROOT/pages/ts-deploy-initialize-fail.adoc [hunk 23]
```diff
@@ -0,0 +1,19 @@
...
+
+* Try redeploying the message flow after 10 minutes.
+
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail.adoc]: see above

File: modules/ROOT/pages/ts-deploy-initialize-fail.adoc [hunk 24]
```diff
@@ -0,0 +1,19 @@
...
+If the error persists, create a support ticket with MuleSoft along with the Reference number in the deployment error.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-deploy-initialize-fail.adoc]:
```suggestion
If the error persists, create a MuleSoft support ticket, including the error reference number.
```

Comment (kbostrom) [reply modules/ROOT/pages/ts-deploy-initialize-fail.adoc]: BREAK HERE

**Review by kbostrom — APPROVED** [review]
Part 1 edit

File: modules/ROOT/pages/ts-failed2deploy-no-capacity.adoc [hunk 25]
```diff
@@ -0,0 +1,17 @@
...
+You need x.x more vCores
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+== Cause
+
+You do not have sufficient vCore capacity to deploy the applications in order to support the message flow.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-failed2deploy-no-capacity.adoc]:
```suggestion
You do not have sufficient vCore capacity to deploy the applications that support the message flow.
```

File: modules/ROOT/pages/ts-failed2deploy-no-capacity.adoc [hunk 26]
```diff
@@ -0,0 +1,17 @@
...
+
+== Solutions
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-failed2deploy-no-capacity.adoc]:
```suggestion
== Solution
```

File: modules/ROOT/pages/ts-failed2deploy-no-capacity.adoc [hunk 27]
```diff
@@ -0,0 +1,17 @@
...
+
+* Contact your MuleSoft Platform Administrator to review the vCore allocation for your business group.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-failed2deploy-no-capacity.adoc:15]:
```suggestion
+ Contact your MuleSoft Platform Administrator to review the vCore allocation for your business group. If vCores are added, wait at least 15 minutes before attempting deployment.
```

File: modules/ROOT/pages/ts-failed2deploy-no-capacity.adoc [hunk 28]
```diff
@@ -0,0 +1,17 @@
...
+
+* Allow 15 minutes after the vCores are added to your business group before redeploying.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-failed2deploy-no-capacity.adoc]: See above; not really two list items but one action and result.

File: modules/ROOT/pages/ts-payload-not-configured.adoc [hunk 29]
```diff
@@ -0,0 +1,22 @@
+= You cannot deploy this message flow because the payload storage is not properly configured
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+== Cause
+
+You have not configured the Payload Storage.
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-payload-not-configured.adoc:10]:
```suggestion
You have not configured payload storage or it is not active.
```

File: modules/ROOT/pages/ts-payload-not-configured.adoc [hunk 30]
```diff
@@ -0,0 +1,22 @@
...
+
+Verify that your Payload Storage is active.
+
+. From within your Anypoint Partner Manager, select your <Hostname> profile.
+
+. Scroll down to the `Payload storage` section and click *Edit*.
+
+. Review your payload storage configuration.
+
+. If not configured, refer to https://docs.mulesoft.com/partner-manager/2.0/setup#setup-the-payload-storage-api[Set up the Payload Storage API].
```
Comment (kbostrom) [inline modules/ROOT/pages/ts-payload-not-configured.adoc]: * If your payload storage is not configured, configure it according to https://docs.mulesoft.com/partner-manager/2.0/setup#setup-the-payload-storage-api[Set Up the Payload Storage API]>
* If your payload storage is configured, verify that it is active:
. From within Anypoint Partner Manager, select your hostname profile.
. Scroll down to *Payload storage* and click *Edit*.
. Review your payload storage configuration and status.

**Review by kbostrom — APPROVED** [review]
Done with all

File: modules/ROOT/pages/troubleshooting.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/troubleshooting.adoc]: Fixed.

File: modules/ROOT/pages/troubleshooting.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/troubleshooting.adoc]: Fixed.

File: modules/ROOT/pages/troubleshooting.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/troubleshooting.adoc]: Fixed.

File: modules/ROOT/pages/troubleshooting.adoc (hunk 4 above)
Comment (mcherem) [reply modules/ROOT/pages/troubleshooting.adoc]: Fixed.

File: modules/ROOT/pages/troubleshooting.adoc (hunk 5 above)
Comment (mcherem) [reply modules/ROOT/pages/troubleshooting.adoc]: Fixed.

File: modules/ROOT/pages/ts-config-deploy.adoc (hunk 6 above)
Comment (mcherem) [reply modules/ROOT/pages/ts-config-deploy.adoc]: Fixed.

File: modules/ROOT/pages/ts-config-deploy.adoc (hunk 7 above)
Comment (mcherem) [reply modules/ROOT/pages/ts-config-deploy.adoc]: Fixed.

File: modules/ROOT/pages/ts-connectivity-system.adoc (hunk 8 above)
Comment (mcherem) [reply modules/ROOT/pages/ts-connectivity-system.adoc]: Fixed.

**Pull request merged**